docs/schema/declarativecontextdefinition-element-solutionmanifestdeclarativeextensions-schem.md

```diff
@@ -46,7 +47,7 @@ The following sections describe attributes, child elements, and parent elements.
 <tbody>
 <tr class="odd">
 <td align="left"><p>**ContentType**</p></td>
-<td align="left"><p>The type name of this context in Outlook 2010. Can be different from the name of the external content type in the model. The value that you specify here affects the name of the Outlook item type as it appears in the Outlook object model.</p><p>In Outlook, the name is appended to the Outlook item type; the full name of an external content type is **Outlook.<OutlookItemType>.<ContentType>**. For example, a **Customer** external type could be **Outlook.Contact.Customer** if it is visualized as an Outlook Contact.</p><p>If the external content type is visualized in a taskpane in Outlook, the full name is **EntityView.<ContentType>**. For example, **EntityView.Customer** indicates how a customer would be visualized in the taskpane. If you use the object model to work on Outlook items, this is the naming scheme that you use to identify your Outlook items.</p>
+<td align="left"><p>The type name of this context in Outlook 2010. Can be different from the name of the external content type in the model. The value that you specify here affects the name of the Outlook item type as it appears in the Outlook object model.</p><p>In Outlook, the name is appended to the Outlook item type; the full name of an external content type is **Outlook. \&ltOutlookItemType\&gt .\&ltContentType\&gt**. For example, a **Customer** external type could be **Outlook.Contact.Customer** if it is visualized as an Outlook Contact.</p><p>If the external content type is visualized in a taskpane in Outlook, the full name is **EntityView.\&ltContentType\&gt**. For example, **EntityView.Customer** indicates how a customer would be visualized in the taskpane. If you use the object model to work on Outlook items, this is the naming scheme that you use to identify your Outlook items.</p>
```

Comment:
This one is pretty tricky, good solution to the problem, but let's leave out the slashes and space.  So for the first fix, it should look like this:
`**Outlook.&ltOutlookItemType&gt.&ltContentType&gt**.`

The 2nd fix:
**EntityView.&ltContentType&gt**